src/components/Paginate.tsx

```diff
@@ -0,0 +1,197 @@
+                onClick={() => {
+                  setCurrentPage(2);
+                }}
+                className={`border duration-150 ease-in-out hover:bg-secondary hover:border-secondary hover:text-primary inline-flex px-4 py-2 transition ${
+                  currentPage === 2 &&
+                  "bg-secondary border-secondary text-primary"
+                }`}
+              >
+                2
+              </button>
+              <button
+                onClick={() => setCurrentPage(3)}
+                className={`border duration-150 ease-in-out hover:bg-secondary hover:border-secondary hover:text-primary inline-flex px-4 py-2 transition ${
+                  currentPage === 3 &&
+                  "bg-secondary border-secondary text-primary"
+                }`}
+              >
+                3
+              </button>
+              <span className={"border inline-flex px-4 py-2"}>...</span>
+              <button
+                onClick={() => setCurrentPage(numberOfPages - 2)}
+                className={`border duration-150 ease-in-out hover:bg-secondary hover:border-secondary hover:text-primary inline-flex px-4 py-2 transition ${
+                  currentPage === numberOfPages - 2 &&
+                  "bg-secondary border-secondary text-primary"
+                }`}
+              >
+                {numberOfPages - 2}
+              </button>
+              <button
+                onClick={() => setCurrentPage(numberOfPages - 1)}
+                className={`border duration-150 ease-in-out hover:bg-secondary hover:border-secondary hover:text-primary inline-flex px-4 py-2 transition ${
+                  currentPage === numberOfPages - 1 &&
+                  "bg-secondary border-secondary text-primary"
+                }`}
+              >
+                {numberOfPages - 1}
+              </button>
+              <button
+                onClick={() => setCurrentPage(numberOfPages)}
+                className={`border duration-150 ease-in-out hover:bg-secondary hover:border-secondary hover:text-primary inline-flex px-4 py-2 transition ${
+                  currentPage === numberOfPages && // If we're on the last page, make the righthand side of
+                  "bg-secondary border-secondary rounded-r-md text-primary" // the last page's button rounded.
+                }`}
+              >
+                {numberOfPages}
+              </button>
+              {currentPage !== numberOfPages && (
+                <button // Only show the next page button if we're not on the last page.
+                  data-e2e-id="next-page-button"
+                  className="inline-flex px-2 py-2 border hover:bg-secondary hover:border-secondary hover:text-primary transition duration-150 ease-in-out"
+                  onClick={() => setCurrentPage(currentPage + 1)}
+                >
+                  <span className="sr-only">Next</span>
+                  <IonIcon className="h-5 w-5" icon={playForwardSharp} />
```

Comment:
I like the general idea of trying to match the theme of playing. But I think this might confuse most people since playback buttons are only really used in UI components involving playback.

To revert the commit that made this change, try using git's revert subcommand

Reply:
That makes sense. I thought it might throw people off, too, but I thought it was somewhat clever and creative, so I tried it out, anyway.

I reverted this commit and ba9a03c44058bf90ba7bb275460d4b2d892b9e28, which is the commit before this one, since the only reason I added buttons for jumping to the first and last pages was so I could display all of the forward and back playback buttons, instead of just a couple of forward and back buttons, which might look weird by themselves.

The first and last page buttons are always visible, anyway, so we'll still be able to access those pages with a single click.

Thank you for your input!